### PR TITLE
Feature/fetchmany generator

### PR DESCRIPTION
**Pull request by mjsqu**

### CODE REVIEW

File: tap_db2/sync_strategies/common.py [hunk 1]
```diff
@@ -232,28 +237,28 @@ def sync_query(
             md_map = metadata.to_map(catalog_entry.metadata)
             stream_metadata = md_map.get((), {})
             replication_method = stream_metadata.get("replication-method")
-
+
```
Comment (s7clarke10) [inline tap_db2/sync_strategies/common.py:240]: Looks like there is a bit of whitespace added to a few of these lines

File: tap_db2/__init__.py [hunk 2]
```diff
@@ -1,10 +1,12 @@
 # import datetime
+from __future__ import generators
```
Comment (s7clarke10) [inline tap_db2/__init__.py:2]: I don't believe you need this lines. It is only required for Python 2.x which we don't support anyway. I didn't use this line in my implementation. Might we worth double checking my understanding of this though.

**Review by s7clarke10 — CHANGES_REQUESTED** [review]
A couple of small considerations.

File: tap_db2/sync_strategies/common.py (hunk 1 above)
Comment (mjsqu) [reply tap_db2/sync_strategies/common.py]: This is now resolved

**Pull request merged**